### DISCUSSION TITLE
Retron 5 board

### DISCUSSION THREAD

**Discussion by Korori03** [sanni/cartreader#318] [Ideas & Suggestions]
Curious if I can use the board from a retron 5. The main board on it blew and the rest is fine. Just curious if someone did a mash up with sanni concept and retron5.

**Comment by sanni** [comment]
You could redesign the Cart Reader PCB so that the spacing between the slots matches the Retron shell and then replace the PCB.
Or you could handwire the slots to an Arduino, but that would take ages and is prone to errors.
